### ISSUE TITLE
Patrons display their preferences

### ISSUE THREAD

**Issue by LanceBailey** [FortWorthGameDev/MayByMoonlight#39]
Depends on #37 

Take a filled glass image created, and cause that image to appear on a thought bubble/speech bubble relative to the patron once they reach about halfway across the bar.

Reference:
https://docs.google.com/document/d/1YmSmHOAYKzmCU9hBz_QsyhOy-GWBxBwKasGI5-ZyYhw/edit#bookmark=id.rbp1oed1bihg

You could get started without #37 if just an empty bubble appeared when they reached a certain point.

**Comment by LanceBailey** [comment]
Patrons now automatically display their drink preferences if there is a 2D Drink Prefab attached to them.  The functionality of the drink only appearing halfway is not yet implemented, creating a new issue related to that (#58).

PR #57
